components/micropython/mpconfigport.h

```diff
@@ -37,12 +34,21 @@
 #define MICROPY_PY_BUILTINS_MEMORYVIEW (1)
 #define MICROPY_PY_RE (1)
 #define MICROPY_PY_DEFLATE (1)
-#define MICROPY_PY_FRAMEBUF (1)
 #define MICROPY_PY_UCTYPES (1)
 #define MICROPY_LONGINT_IMPL MICROPY_LONGINT_IMPL_MPZ
 #define MICROPY_PY_JSON (1)
 #define MICROPY_PY_IO_IOBASE (1)
 
+#ifdef ENABLE_I2C
```

Comment:
Should we do something more specific to make it obvious what the flags are, something like `MICROPY_PORT_ENABLE_I2C`?

Reply:
Discussed offline that it's fine for now.